dashboard/app/serializers/api/v1/pd/application_serializer.rb

```diff
@@ -0,0 +1,7 @@
+class Api::V1::Pd::ApplicationSerializer < ActiveModel::Serializer
+  attributes :regional_partner, :locked?, :notes, :form_data
+
+  def regional_partner
+    RegionalPartner.find(object.regional_partner_id).name
```

Comment:
[regional_partner](https://github.com/code-dot-org/code-dot-org/blob/b359bdf0ca7679f6162ac59b46bccb58457dfecc/dashboard/app/models/pd/application/application_base.rb#L65) is already an association on the model, so this can be simplified...

I'd suggest calling it `regional_partner_name` and either providing that directly on the model, or here as  `object.regional_partner.try(:name)`.

Make sure to handle partners - not all applications have a matching partner.